Source/Provers/SMTLib/SMTLibLineariser.cs

```diff
@@ -322,8 +322,21 @@ public bool Visit(VCExprQuantifier node, LineariserOptions options)
       UnderQuantifier++;
       Namer.PushScope(); try {
 
-        string kind = node.Quan == Quantifier.ALL ? "forall" : "exists";
-        wr.Write("({0} (", kind);
+        VCQuantifierInfos infos = node.Infos;
+        if (infos.isFunctionDefinition) {
+	  wr.Write("(define-fun ");
+	  var naryExpr = node.Body as VCExprNAry;
+	  Contract.Assert(naryExpr != null);
+	  var funCall = naryExpr[0] as VCExprNAry;
+	  Contract.Assert(funCall != null);
+	  VCExprBoogieFunctionOp op = (VCExprBoogieFunctionOp)funCall.Op;
```

Comment:
indent

Reply:
Oops - my emacs was set up to insert tabs instead of spaces.  Fixed.